Lib/fontTools/feaLib/ast.py

```diff
@@ -694,15 +820,17 @@ def asFea(self, indent=""):
 
 
 class LanguageStatement(Statement):
+    """A ``language`` statement within a feature."""
     def __init__(self, language, include_default=True, required=False,
                  location=None):
         Statement.__init__(self, location)
         assert(len(language) == 4)
-        self.language = language
-        self.include_default = include_default
+        self.language = language #: A four-character language tag
```

Comment:
here and elsewhere, please separate inline comments with two spaces

```suggestion
        self.language = language  #: A four-character language tag
```